### ISSUE TITLE
SSH Node with private key + passphrase is not working

### ISSUE THREAD

**Issue by wiinxt** [n8n-io/n8n#1950]
**Describe the bug**
I tried to use an SSH node with a private key + a passphrase but I have this error :
"ERROR: Cannot parse privateKey: Encrypted private OpenSSH key detected, but no passphrase given"
My passphrase field is however well filled.

Stack
```
Error: Cannot parse privateKey: Encrypted private OpenSSH key detected, but no passphrase given
    at Client.connect (/usr/local/lib/node_modules/n8n/node_modules/ssh2/lib/client.js:243:13)
    at /usr/local/lib/node_modules/n8n/node_modules/node-ssh/lib/cjs/index.js:193:24
    at new Promise (<anonymous>)
    at NodeSSH.connect (/usr/local/lib/node_modules/n8n/node_modules/node-ssh/lib/cjs/index.js:173:15)
    at async Object.execute (/usr/local/lib/node_modules/n8n/node_modules/n8n-nodes-base/dist/nodes/Ssh/Ssh.node.js:300:17)
    at async /usr/local/lib/node_modules/n8n/node_modules/n8n-core/dist/src/WorkflowExecute.js:424:47
```

**To Reproduce**
1. Create a SSH Credentials with a private key and a passphrase
2. Create a SSH node
3. Use the SSH Credentials created in step 1
4. Try to execute Node

**Expected behavior**
I expect the connection to be established with the SSH server.

**Environment (please complete the following information):**
 - OS: debian 10
 - n8n Version: 0.125.0 (docker image)
 - Node.js Version: v14.15.5 (docker image)
 - Database system: SQLite
 - Operation mode [e.g. own; operation modes are `own`, `main` and `queue`. Default is `own`]: own

**Additional context**
I have checked the code in Ssh.node.js and I think the "not" in the if condition should not be there, when I delete it locally it works fine.
https://github.com/n8n-io/n8n/blob/320e5f9f970a508bdce1185d3a71884f2c6cec90/packages/nodes-base/nodes/Ssh/Ssh.node.ts#L323:L325
Maybe I'm wrong, idk ¯\\\_(ツ)_/¯

**Comment by RicardoE105** [comment]
@wiinxt the issue got fixed. We will let you know when it's released.

https://github.com/n8n-io/n8n/pull/1966

**Comment by janober** [comment]
Fix got released with n8n@0.127.0

**Comment by wiinxt** [comment]
Thanks !
